granulate_utils/metadata/cloud.py

```diff
@@ -103,6 +107,13 @@ def get_gcp_metadata() -> Optional[GcpInstanceMetadata]:
     )
 
 
+def get_gcp_region_from_zone(zone: str) -> str:
+    zone = zone.rsplit('/', 1)[-1]
```

Comment:
What's the operation you are doing here? How does it look exactly? Give a sample comment on input/output when doing string manipulations

Reply:
An example of zone would be: `projects/234/zones/us-east1-a`
So,
`get_gcp_region_from_zone('projects/234/zones/us-east1-a')` -> returns
`us-east1`

Reply:
Comment in code pls